python/lancedb/fts.py

```diff
@@ -113,6 +118,14 @@ def populate_index(index: tantivy.Index, table: LanceTable, fields: List[str]) -
     return row_id
 
 
+def open_writer(index: tantivy.Index, writer_heap_size: int = None) -> "IndexWriter":
+    if writer_heap_size is not None:
+        writer = index.writer(heap_size=writer_heap_size)
+    else:
+        writer = index.writer(heap_size=1024 * 1024 * 1024)
+    return writer
```

Comment:
Why are we burying the default? It's less code and better documented if we just make the default `1024 * 1024 * 1024` instead of `None`.

Reply:
added as parameter defaults